.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+compose.yml
```

Comment:
OK just got time to review this PR.

So I would suggest we remove docker related files in this PR and instead making all docker related files in https://github.com/yamlresume/yamlresume/pull/5.

So `.dockerignore`, `Dockerfile` and `compose.yml` is not needed in this PR.